lib/evm/src/client.rs

```diff
@@ -54,29 +54,4 @@ impl EVMClient for BasinClient {
             .await
             .map_err(|e| Error::Evm(e.to_string()))
     }
-
-    async fn deals(
-        &self,
-        publication: &str,
-        offset: u64,
-        limit: u32,
-    ) -> Result<Vec<DealInfo>, Error> {
-        self.contract
-            .paginated_deals(
```

Comment:
removes the old contract functions and variables: `DealInfo`, `paginatedDeal`, `latest_n_deals`